src/doom/g_game.c

```diff
@@ -1184,7 +1208,12 @@ void G_Ticker (void)
 	    break; 
 	} 
     }
-    
+
+    // [crispy] demo sync of revenant tracers and RNG (from prboom-plus)
+    if (paused & 2 || (!demoplayback && menuactive && !netgame))
```

Comment:
So, all this ominous `paused & 2` mode does is catching up `demostarttics` with `gametics` while not actually processing tics?

Reply:
I must admit that I'm rather confident that I could give a reasonably satisfactory answer to this question if I knew what `paused & 2` really meant and why it was used instead of the much more straightforward 'paused'.